metadata-load/pom.xml

```diff
@@ -30,11 +30,81 @@
             <version>1.5.48</version>
             <type>jar</type>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-jdbc</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.25</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.7.25</version>
+        </dependency>
     </dependencies>
 
+    <profiles>
+        <profile>
+            <id>Non-Travis</id>
+            <dependencies>
+                <dependency>
+                    <groupId>com.oracle.jdbc</groupId>
+                    <artifactId>ojdbc8</artifactId>
+                    <version>12.2.0.1</version>
+                </dependency>
+                <dependency>
+                    <groupId>com.oracle.jdbc</groupId>
+                    <artifactId>xdb6</artifactId>
+                    <version>12.2.0.1</version>
+                </dependency>
+                <dependency>
+                    <groupId>com.oracle.xml</groupId>
+                    <artifactId>xmlparserv2</artifactId>
```

Comment:
Xmlparserv2 doesn't seem to be needed anymore on version 12.2.0.1, plus it is impossible to download that version from the oracle maven repository, I remember reading a while ago that it was only meant for retrocompatibility. As far as I am aware the tests still pass without it.

Reply:
When i remove it i am getting java.lang.NoClassDefFoundError: oracle/xml/binxml/BinXMLMetadataProvider; will explore and discuss with you